### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by SajaLahaleeh**

### CODE REVIEW

**Review by someyaaltous — APPROVED** [review]

**Review by DuaaH — APPROVED** [review]

**Comment by DuaaH** [comment]
me and saja make this part from readme

**Review by Muniralsharif — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1 +1,21 @@
...
\ No newline at end of file
+# Football_teams
+
+### About :
+
+our website allows user to search about footbal teams in specific country ,displays the teams in this country, and some details about it.
+
+### User journey:
+
+- As a user I should be able to enter a country name.
```
Comment (jema28) [inline README.md]: You can simplify this to:

As a user, I will be able to:
- enter a country name
- click on search button and see the all teams in this country.
- click on team name and see more details about them.

File: README.md [hunk 2]
```diff
@@ -1 +1,21 @@
-# Sport
\ No newline at end of file
+# Football_teams
+
+### About :
+
+our website allows user to search about footbal teams in specific country ,displays the teams in this country, and some details about it.
```
Comment (jema28) [inline README.md]: A few small grammatical mistakes: 'football', comma in the wrong place ',displays'

File: README.md [hunk 3]
```diff
@@ -1 +1,21 @@
...
+
+### User journey:
+
+- As a user I should be able to enter a country name.
+- As a user I should be able to click on button search, then I see the all of teams in this country.
+- As a user I should be able to click on team name, then I see more details about selected team.
+
+
+### Skitch :
```
Comment (jema28) [inline README.md]: Spelling error here: 'Sketch'

**Review by jema28 — CHANGES_REQUESTED** [review]

**Comment by SajaLahaleeh** [comment]
we solved the conflict @jema28

**Review by jema28 — APPROVED** [review]

**Comment by jema28** [comment]
@SajaLahaleeh Just a note on commit messages, it's good to format them like this:
'add file structure to the readme' - the verb in this case add should be in present tense and avoid 'I' in the message. Does this make sense?

**Review by jema28 — APPROVED** [review]

**Pull request merged**